### ISSUE TITLE
Can i use them for nw ?

### ISSUE THREAD

**Issue by ipostol** [zalmoxisus/remote-redux-devtools#29]
Hello, i am have some redux project, and our production started on NW. Can i use remote-devtools for them? Now i am try do it but it don`t work :(

**Comment by zalmoxisus** [comment]
Yes, it should work even in node environment. You can just set `realtime` parameter to `true` and it will work. Just make sure not to have it in production, as it will affect the performance.

**Comment by ipostol** [comment]
I will use them in pre-production. For debug in nw. 
I am have next code in prod env:
```js
  store = compose(
    applyMiddleware(clientMiddleware(socket), promiseMiddleware),
    reduxReactRouter({
      getRoutes,
      createHistory
    }),
    persistState(window.location.href.match(/[?&]debug_session=([^&]+)\b/)),
    devTools({
      name: `My name`, realtime: true
    })
  )(createStore)(reducers);
```

If i am start this in browser all be ok, mb sometime chrome extension not found application.
But if i am start this in NW chrome extension not found anything.

**Comment by zalmoxisus** [comment]
I'm using it successfully on Electron. If you publish a repo I could run, I'll take a look what blocks sending actions there.

**Comment by ipostol** [comment]
I will do it later, course now i haven`t open source code. Thank you for response.
